backend.native/tests/interop/objc/msg_send/messaging.h

```diff
@@ -0,0 +1,47 @@
+#import <objc/NSObject.h>
+
+@interface PrimitiveTestSubject : NSObject
+
++ (int)intFn;
++ (float)floatFn;
++ (double)doubleFn;
+
+@end;
+
+typedef struct {
+    float f;
+} SingleFloat;
+
+typedef struct __attribute__((packed)) {
+    char f1;
+    short f2;
+    char f3;
+    char f4;
+} SimplePacked;
```

Comment:
Maybe add this struct too?
https://github.com/JetBrains/kotlin-native/pull/3323#discussion_r322597676